src/Adapter/Product/AdminProductDataUpdater.php

```diff
@@ -80,6 +80,9 @@ public function activateProductIdList(array $productListId, $activate = true)
             }
             $product->active = ($activate ? 1 : 0);
             $product->update();
+            if (in_array($product->visibility, array('both', 'search')) && Configuration::get('PS_SEARCH_INDEXATION')) {
+                Search::indexation(false, $product->id);
+            }
```

Comment:
@matthieu-rolland 
Only this modification is directly linked to this PR.

Reply:
Ok, this looks fine, I will approve your pull request once you have removed the hook in the classic template 👍

Reply:
Hello @PeeyushAgrawalWebkul  !
Do you think you can remove the hook so I can approve your PR? If you can't I can take if from there if you want.

Reply:
Hello @matthieu-rolland 

Thanks,
Yes, Please proceed with the branch 
PeeyushAgrawalWebkul:solution-#9663

Reply:
Hello @matthieu-rolland 

I have removed the hook. Please check and proceed to merge the PR.